init-mongo.js

```diff
@@ -0,0 +1,14 @@
+db.auth('root', 'rootpassword')
+
+db = db.getSiblingDB('star_tides')
+
+db.createUser({
+  user: 'evan',
```

Comment:
Can you parameterize this and write some docs for this please?

Reply:
Deleted